repos/fdbt-site/src/design/modules/_helpers.scss

```diff
@@ -0,0 +1,7 @@
+.govuk-flex {
```

Comment:
Should we have govuk on non-gov styles? Might be clearer to not namespace our own custom styles?